js/client-teambuilder.js

```diff
@@ -347,7 +347,7 @@
 				buf += '<h2>Hi</h2>';
 				buf += '<p>Did you have a good day?</p>';
 				buf += '<p><button class="button" name="greeting" value="Y"><i class="fa fa-smile-o"></i> Yes, my day was pretty good</button> <button class="button" name="greeting" value="N"><i class="fa fa-frown-o"></i> No, it wasn\'t great</button></p>';
-				buf += '<h2>All teams</h2>';
+				buf += '<h2>All teams (' + teams.length + ')</h2>';
```

Comment:
This should be in `<small>`, and probably `<small>` should be styled not to be bold. Otherwise, good idea!